.github/workflows/ci-validation.yml

```diff
@@ -35,8 +35,15 @@ jobs:
 
     - name: Build (Ubuntu 20.04)
       run: |
-        tools/ci-build.sh
+        tools/ci-build.sh || echo "BUILD_SUCCESS=false" >> $GITHUB_ENV
```

Comment:
Why is it not sufficient to fail normally?

Reply:
This tool is not part of the P4C repository, if it does not build it should not block a PR. It must be fixed separately.

Reply:
I believe most of the tests being non-required is an accident. We do not have control over adjusting the settings which of the tests are required. We should fix this. 

I am okay with removing this check, I maintain the tool after all. 

@mbudiu-vmw Should we make this a non-required test?